src/test/java/seedu/address/logic/parser/DeleteTaskComParserTest.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.DeleteTaskCommand;
+
+class DeleteTaskComParserTest {
```

Comment:
Would it be better to name this file `DeleteTaskCommandParserTest` instead, so that it follows the naming convention of all other files as well.